apps/frontend/src/pages/Login.tsx

```diff
@@ -1,5 +1,90 @@
+import { useState, FormEvent } from "react";
+import { Box, Button, TextField, Typography, Link } from "@mui/material";
+import martletPlaceLogo from "../images/martletplace-logo.png";
+import { useNavigate } from "react-router-dom";
+import { useStyles } from "../styles/pageStyles";
+import axios from "axios";
+
 const Login = () => {
-  return <h1>Login</h1>;
+  const classes = useStyles();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+  const navigate = useNavigate();
+  const isFormIncomplete = !email || !password;
+
+  const handleLogin = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    try {
+      const response = await axios.post("/api/login", {
+        email,
+        password,
+      });
+      if (response.status === 201) {
+        navigate("/");
+      }
+    } catch (error) {
+      console.error("Login failed:", error);
+      setError("Login unsuccessful. Invalid username and password combination");
+    }
```

Comment:
Same thing as before about using [Axios' error handling](https://axios-http.com/docs/post_example), but if you think it won't cause any issues we can leave it.